atlasdb-impl-shared/src/main/java/com/palantir/atlasdb/sweep/priority/SweepPriorityCalculator.java

```diff
@@ -107,6 +117,53 @@ public SweepPriorityCalculator(KeyValueService kvs, SweepPriorityStore sweepPrio
         return scores;
     }
 
+    private boolean decideWhetherToLogAllPriorities() {
+        ZonedDateTime now = ZonedDateTime.now();
```

Comment:
We could use a rate-limiter that has `1 permit/(10 minutes)` or `0.1 permit/minute`.

Reply:
Is this required for merging?

Reply:
Not really required, but it will be cleaner if we use the Java rate-limiter, we have this style of rate-limiting logging at multiple places in the codebase and this one will be a snowflake, also we are implementing custom logic for exactly what a ratelimiter does.